test/preview/components/embedders/test_gradient_document_embedder.py

```diff
@@ -0,0 +1,150 @@
+import pytest
+from haystack.preview.components.embedders.gradient_document_embedder import GradientDocumentEmbedder
+from unittest.mock import MagicMock, NonCallableMagicMock
+import numpy as np
+
+from haystack.preview import Document
+
+
+access_token = "access_token"
+workspace_id = "workspace_id"
+model = "bge-large"
+
+
+def has_gradient():
+    try:
+        import gradientai
+
+        return True
+    except ModuleNotFoundError:
+        return False
+
+
+@pytest.mark.skipif(not has_gradient(), reason="Gradient is not installed")
```

Comment:
No need for this condition, if `gradientai` is not installed these tests should fail.

Reply:
done